docs/json-schema/comment.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$id": "https://sublinks.org/comment.schema.json",
+  "title": "Comment",
+  "description": "A comment in a Sublinks Post attributable to an author.",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The unique identifier for the comment.",
+      "type": "string"
+    },
+    "post_id": {
+      "description": "The unique identifier of Post comment is for.",
+      "type": "string"
+    },
+    "content": {
+      "description": "The content of the comment.",
+      "type": "string"
+    },
+    "author": {
+      "description": "The author of the comment.",
```

Comment:
```suggestion
      "description": "The unique identifier for the actor that is the author of the comment.",
```

Reply:
Maybe? idk. I just think it should be more representative that it is the federated id of the user, eg https://discuss.online/u/lazyguru